### ISSUE TITLE
Normalizer doesn't support multiple event loops

### ISSUE THREAD

**Issue by OvervCW** [gmr/email-normalize#5]
`Normalizer.normalize()` breaks when it is used in a program that contains multiple event loops, for example because it's multithreaded. A common example where this might be the case is Django + uvicorn.

Example:

```python
import asyncio
import threading
import time

import email_normalize

def fn():
    asyncio.set_event_loop(asyncio.new_event_loop())

    print(email_normalize.normalize("f.o.o+bar@gmail.com"))

t1 = threading.Thread(target=fn)
t2 = threading.Thread(target=fn)

t1.start()

time.sleep(1)

t2.start()

t1.join()
t2.join()
```

Output:

```
Result(address='f.o.o+bar@gmail.com', normalized_address='foo@gmail.com', mx_records=[(5, 'gmail-smtp-in.l.google.com'), (10, 'alt1.gmail-smtp-in.l.google.com'), (20, 'alt2.gmail-smtp-in.l.google.com'), (30, 'alt3.gmail-smtp-in.l.google.com'), (40, 'alt4.gmail-smtp-in.l.google.com')], mailbox_provider='Google')
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "reproduce.py", line 10, in fn
    print(email_normalize.normalize("f.o.o+bar@gmail.com"))
  File "<<xxx/site-packages>>/email_normalize/__init__.py", line 259, in normalize
    return loop.run_until_complete(normalizer.normalize(email_address))
  File "/usr/lib/python3.7/asyncio/base_events.py", line 579, in run_until_complete
    return future.result()
  File "<xxx/site-packages>/email_normalize/__init__.py", line 192, in normalize
    mx_records = await self.mx_records(domain_part)
  File "<xxx/site-packages>/email_normalize/__init__.py", line 153, in mx_records
    records = await self._resolver.query(domain_part, 'MX')
RuntimeError: Task <Task pending coro=<Normalizer.normalize() running at <xxx/site-packages>/email_normalize/__init__.py:192> cb=[_run_until_complete_cb() at /usr/lib/python3.7/asyncio/base_events.py:153]> got Future <Future pending> attached to a different loop
```

This is caused by the fact that a single `aiodns.DNSResolver` is being shared across event loops due to it [being a singleton](https://github.com/gmr/email-normalize/blob/master/email_normalize/__init__.py#L135).

For now, I've worked around it in my own project by doing this:

```python
async def normalize_email(email_address: str) -> str:
    normalizer = email_normalize.Normalizer()
    email_normalize.Normalizer._instance = None
    return (await normalizer.normalize(email_address)).normalized_address
```

**Comment by gmr** [comment]
Interesting - I'll revisit the singleton behavior as that's a pain. I'm pretty sure I did that for performance reasons, but will double check.

**Comment by gmr** [comment]
Addressed in 2.0

**Comment by OvervCW** [comment]
Many thanks!
